public/assets/logos/LogoFacebook.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
Is this component used?

Reply:
oh I added it and didnt use it. I will do this in a later work. Thanks I will remove it.